src/settings.py

```diff
@@ -41,6 +41,9 @@ class GlobalSettings:
     repeated_nextto_num_dots = 25
 
     # painting env parameters
+    painting_initial_holding_prob = 0.0
+    painting_num_objs_train = [3, 4]
```

Comment:
note we won't be able to override this with command line args lol, oh well

Reply:
good point... yeah lol oh well is all I have to say too